core/src/main/java/org/apache/calcite/sql/SqlJdbcFunctionCall.java

```diff
@@ -732,6 +732,7 @@ private JdbcToInternalLookupTable() {
       map.put("MONTH", simple(SqlStdOperatorTable.MONTH));
       map.put("WEEK", simple(SqlStdOperatorTable.WEEK));
       map.put("DAYOFYEAR", simple(SqlStdOperatorTable.DAYOFYEAR));
+      map.put("DAY", simple(SqlStdOperatorTable.DAYOFMONTH));
```

Comment:
How is it different from `DAYOFMONTH`?

Reply:
Thanks for your review. It is same as `DAYOFMONTH `, but it is not a valid JDBC function. I've deleted it.